examples/python/hori_diff.py

```diff
@@ -0,0 +1,151 @@
+"""Horizontal diffusion stencil HIR generator
+
+This program creates the HIR corresponding to an horizontal diffusion stencil using the Python API of the HIR.
+The horizontal diffusion is a basic example that contains horizontal data dependencies that need to be resolved 
+by the compiler passes. 
+The code is meant as an example for high-level DSLs that could generate HIR from their own 
+internal IR. 
+The program contains two parts: 
+    1. construct the HIR of the example
+    2. pass the HIR to the dawn compiler in order to run all optimizer passes and code generation.
+       In this example the compiler is configured with the CUDA backend, therefore will code generate
+       an optimized CUDA implementation.
```

Comment:
should we specify the PYTHONPATH in this header?

Reply:
I think is a normal thing to expect (unless you go via pip) that for compiled and installed modules one has to set the PYTHONPATH to the directory of the module, isnt? 
So I would not repeat it in all examples... but we could put it in a README or general documentation of how to use python

Reply:
I would probably just write a couple of lines somewhere. I agree repreating it everywhere might be suboptimal